lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -601,28 +611,26 @@ export default class BitcoinProcessor {
     const endBlockHeight = await this.bitcoinClient.getCurrentBlockHeight();
     console.info(`Processing transactions from ${startBlockHeight} to ${endBlockHeight}`);
 
+    let blockHeight = startBlockHeight;
     let previousBlockHash = startBlock.previousHash;
+    while (blockHeight <= endBlockHeight) {
```

Comment:
Curious: any reason to change from for to while loop? Readability?

Reply:
Yes, just readability. Because there are actually two params being incremented/change every loop, one part of the `for` line and the other at the end of the loop. Changing it make it consistent.